code/game/turfs/closed/minerals.dm

```diff
@@ -312,6 +309,28 @@
 		/obj/item/stack/ore/iron = 95,
 	)
 
+/turf/closed/mineral/random/air
```

Comment:
Why is there a random typepath declaration here, you have another declaration of the same typepath just below?